README.md

```diff
@@ -77,7 +77,14 @@ included in `flymake-languagetool-active-modes`.
 
 ## 🧪 Configuration
 
-### Language
+### Sentence Awareness
+`flymake-languagetool` now uses emacs sentence navigation to send only portions
+of the text that have been modified rather than the entire buffer. In order for
+this to behave in an expected manner, you may have to modify
+`sentence-end-double-space`. The default is set to `true` and may cause some
+contextual issues if you only use a single space to separate sentences.
+
+### language
```

Comment:
Some nits:

- `### language` -> `### Language`

Make there one newline under the title:

```
### Sentence Awareness
| <-- newline
...

### Language
| <-- newline
...
```